administrator/modules/mod_backward/tmpl/default.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  mod_backward
+ *
+ * @copyright   (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Language\Text;
+
+?>
+<?php if ($compat || $compat6) : ?>
+    <div class="header-item-content joomlaversion">
+        <div class="header-item-text no-link">
+            <span class="icon-shield-alt" aria-hidden="true"></span>
+            <span aria-hidden="true"><?php echo Text::_('MOD_BACKWARD_TEXT') . ($compat ? '5' : '') . ($compat && $compat6 ? ', ' : '') . ($compat6 ? '6' : ''); ?></span>
```

Comment:
```suggestion
<?php if ($compat || $compatNext) : ?>
    <div class="header-item-content joomlaversion">
        <div class="header-item-text no-link">
            <span class="icon-shield-alt" aria-hidden="true"></span>
            <span aria-hidden="true"><?php echo Text::_('MOD_BACKWARD_TEXT') . ($compat ? '5' : '') . ($compat && $compatNext ? ', ' : '') . ($compatNext ? '6' : ''); ?></span>
```
More generic name so it does not need to change variable names with every new major release.

Reply:
In addition, it could make sense not to use hard-coded strings `'5'` and `'6'` but the major version from Version.php for this purpose.

Reply:
I will create a PR for your PR.